### PR TITLE
Maze GUI

### PR DESCRIPTION
**Pull request by scohen40**
JComponent not displaying the table correctly. Not sure why.

### CODE REVIEW

File: src/main/java/cohen/maze/mazeGUI/AnimatedMaze.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    protected void paintComponent(Graphics g) {
+        super.paintComponent(g);
+
+        componentHeight = this.getHeight();
+        componentWidth = this.getWidth();
+        seventhHeight = componentHeight/7;
+        seventhWidth = componentWidth/7;
+
+//        g.translate(0, this.getHeight());
+        maze = new Maze(7, 7);
```
Comment (andrewoid) [inline src/main/java/cohen/maze/mazeGUI/AnimatedMaze.java]: do not create a new maze each time

File: src/main/java/cohen/maze/mazeGUI/AnimatedMaze.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+    }
+
+    /**
+     * The paintMaze() method                runs through the generated maze and paints the existing walls.
+     * @param g
+     */
+    void paintMaze(Graphics g) {
+        for (int x = 0; x < maze.getHeight(); x++) {
+            System.out.println("|");
```
Comment (andrewoid) [inline src/main/java/cohen/maze/mazeGUI/AnimatedMaze.java]: You don't need this println

Comment (scohen40) [reply src/main/java/cohen/maze/mazeGUI/AnimatedMaze.java]: Done

File: src/main/java/cohen/maze/mazeGUI/AnimatedMaze.java (hunk 1 above)
Comment (scohen40) [reply src/main/java/cohen/maze/mazeGUI/AnimatedMaze.java]: Done

**Comment by scohen40** [comment]
The project is basically complete. The only thing left is to replace the circle icon with an actual picture. (DONE)

**Comment by scohen40** [comment]
I would also like to change all rows and columns to base 0 and just adjust as needed. Right now it's more confusing than it needs to be.

File: src/main/java/cohen/maze/mazeGUI/AnimatedMaze.java [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+    public AnimatedMaze(Maze maze, Man man) {
+        this.maze = maze;
+        this.man = man;
+    }
+
+    protected void paintComponent(Graphics g) {
+        //having trouble loading.
+        try {
+            manImage = ImageIO.read(AnimatedMaze.class.getResourceAsStream("/shortMan.png"));
```
Comment (andrewoid) [inline src/main/java/cohen/maze/mazeGUI/AnimatedMaze.java:29]: Do this in the constructor so it only happens once, not in paintComponent() where it happens every time the player moves.

File: src/main/java/cohen/maze/mazeGUI/AnimatedMazeGUI.java [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+        animatedMaze = new AnimatedMaze(maze, man);
+        panel.add(animatedMaze, BorderLayout.CENTER);
+
+        addKeyListener(new MyKeyAdapter());
+
+        setContentPane(panel);
+
+    }
+
+    private class MyKeyAdapter extends KeyAdapter {
```
Comment (andrewoid) [inline src/main/java/cohen/maze/mazeGUI/AnimatedMazeGUI.java:33]: Name your classes after what they do, not what they are.

**Pull request merged**